src/SApp.jsx

```diff
@@ -1,7 +1,11 @@
 import styled from 'styled-components';
 
 const SApp = styled.div`
+<<<<<<< HEAD
+  background-color: white;
+=======
   min-height: 100vh;
+>>>>>>> 62455150009ec42eed32f39f015ce4231ac20b8e
```

Comment:
:warning: **!!! THIS PR WAS NOT READ PROPERLY ENOUGH !!!** :warning: 
Git conflict artefacts